projects/plugins/crm/includes/ZeroBSCRM.AJAX.php

```diff
@@ -1345,6 +1345,13 @@ function ZeroBSCRM_get_quote_template() {
 				$quote_date = date( 'd/m/Y', time() );
 			}
 
+			if ( empty( $quote_notes ) ) {
+				$quote_notes = '';
```

Comment:
Two things here... 

If we are checking that `$quote_notes` is empty, do we need to ensure/force it set it as an empty string after the condition?

`$quote_notes = '0';` >> `empty( $quote_notes )` >> `true`

Reply:
It doesn't look like we need to do that actually, thanks! I originally had the empty string as the 'else', then moved it as the negative case to before the if statement, but even though the variable is undefined vs an empty string it all still works. Just added another commit.